ipld/kamt/src/kamt.rs

```diff
@@ -130,6 +130,12 @@ where
     pub fn is_empty(&self) -> bool {
         self.root.is_empty()
     }
+
+    /// Clears all entries in the KAMT and resets the root to an empty node.
+    pub fn clear(&mut self) {
+        self.root = Node::default(); // Reset the root to an empty node
+        self.flushed_cid = None; // Invalidate the flushed CID
```

Comment:
Nit: can we check `is_empty` first to avoid writing the root if we're already empty?

Reply:
updated 04fee90f